.cminx-config.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023 CMakePP
```

Comment:
Where is this file being used? I checked the master action for deploy docs, and I don't see a `-s` parameter being passed to CMinx:

https://github.com/CMakePP/.github/blob/9ff017d38c8d1b41ce61ba0534fa8a84496d3117/.github/workflows/deploy_docs_master.yaml#L95-L98

Reply:
Opened an issue: CMakePP/.github#22

Reply:
Thanks for catching this. I had a `-s` in my local script, but must have forgotten it in the action.